src/components/registration/index.js

```diff
@@ -7,6 +7,25 @@ import Checkbox from './Checkbox';
 import FormButton from './FormButton';
 
 const Registration = () => {
+  const [state, setState] = useState({
+    password: { value: '', isValid: false },
+    passwordRepeat: { value: '', isValid: false },
+    email: { value: '', isValid: false },
+    termsAccepted: false
+  });
+
+  const onTextChanged = name => ({ target }) => {
+    const { validity, value } = target;
+    const isValid = validity.patternMismatch || validity.typeMismatch;
+    if (name === 'password') {
+      setState({
+        ...state,
+        [name]: { value, isValid },
+        passwordRepeat: { isValid: state.passwordRepeat.value === state.password.value }
+      });
```

Comment:
Coś tu chyba nie działa tak jak powinno, bo znika od razu po wprowadzeniu następnego klawisza. Wydaje mi się, że tak jak było, to było okej.

Reply:
właśnie wcześniej przy zmianie powtórnego hasła, nie pojawiał się komunikat, że hasła są różne

Reply:
No nie wiem, wszystko działało raczej bo tam jest pattern też.

Reply:
tylko właśnie u mnie się nie aktualizowało - pattern jest sprawdzany do tooltipa przy zmianie tekstu danego komponentu, jak zmienia się tekst w password, to w passwordRepeat nie ma aktualizacji isValid

Reply:
![obraz](https://user-images.githubusercontent.com/22578403/76526005-28fb6b80-646d-11ea-9b77-a94912795b81.png)
![obraz](https://user-images.githubusercontent.com/22578403/76526030-331d6a00-646d-11ea-8bf3-67709ea13f75.png)

Reply:
Przeniosłem logikę tego bezpośrednio do komponentu